src/locale/locales/ca/messages.po

```diff
@@ -8557,7 +8557,7 @@ msgstr "La política de privacitat ha estat traslladada a <0/>"
 
 #: src/view/com/composer/state/video.ts:395
 msgid "The selected video is larger than 50MB."
-msgstr "El vídeo triat és més gran de 50MB."
+msgstr "El vídeo triat és més gran de 50 MB."
```

Comment:
```suggestion
msgstr "El vídeo triat és més gran de 50 MB."
```
tiny nit: it might be good to use a non-breaking space character here so that `50 MB` is always displayed together (in case there's a line break).

Reply:
Thanks, I'll leave it as it is, in catalan the rule is to have a space between the number and the units https://community.languagetool.org/rule/show/ESPAI_EN_UNITATS?lang=ca&subId=2